changelog/22.0/22.0.0/summary.md

```diff
@@ -315,21 +313,20 @@ never be promoted, neither by planned nor by emergency reparents.
 
 ### <a id="semi-sync-monitor"/>Semi-sync monitor in vttablet</a>
 
-A new component has been added to the vttablet binary to monitor the semi-sync status of primary vttablets.
+Primary VTTablets now monitor the semi-sync status of their underlying MySQL instance .
 We've observed cases where a brief network disruption can cause the primary to get stuck indefinitely waiting for semi-sync ACKs.
-In rare scenarios, this can block reparent operations and render the primary unresponsive.
+In rare scenarios, this can also block reparent operations and render the primary unresponsive.
 More information can be found in the issues [#17709](https://github.com/vitessio/vitess/issues/17709) and [#17749](https://github.com/vitessio/vitess/issues/17749).
 
-To address this, the new component continuously monitors the semi-sync status. If the primary becomes stuck on semi-sync ACKs, it generates writes to unblock it. If this fails, VTOrc is notified of the issue and initiates an emergency reparent operation.
+To address this, the primary VTTablets continuously monitor their semi-sync status. If the primary MySQL gets stuck waiting for semi-sync ACKs, the monitor generates synthetic writes to unblock it. If this fails, VTOrc is notified of the issue and it will initiate an emergency reparent operation.
 
 The monitoring interval can be adjusted using the `--semi-sync-monitor-interval` flag, which defaults to 10 seconds.
 
 ---
 
 ### <a id="new-errors-fatal-tx"/>Wrapped fatal transaction errors</a>
 
-
-When a query fails while being in a transaction, due to the transaction no longer being valid (e.g. PRS, rollout, primary down, etc), the original error is now wrapped around a `VT15001` error.
+When a query fails while running in a transaction, due to the transaction no longer being valid (e.g. PRS, rollout, primary down, etc.), the original error is now wrapped in a `VT15001` error.
 
 For non-transactional queries that produce a `VT15001`, VTGate will try to rollback and clear the transaction.
```

Comment:
@frouioui this is not making sense. Why does it say "non-transactional"? What does it mean to rollback and clear the transaction for a non-transactional query?

Reply:
I meant statements that are not `BEGIN`, `COMMIT`, `ROLLBACK` etc

Reply:
Hmm.. calling them non-transactional is confusing in this context. Let's just say "When a query produces a `VT15001`....". The nuance around COMMIT/ROLLBACK/BEGIN doesn't need to be in the release notes.

Reply:
fixed via 8498f40267